src/Notifier.js

```diff
@@ -80,10 +80,11 @@ const Notifier = {
             if (ev.getContent().body) msg = ev.getContent().body;
         }
 
-        const avatarUrl = ev.sender ? Avatar.avatarUrlForMember(
-            ev.sender, 40, 40, 'crop'
-        ) : null;
+        if (!this.isBodyEnabled()) {
+            msg = '';
+        }
```

Comment:
Surely blindly setting the message to the empty string is not what we want here? I'd have thought it should at least read 'message' (or 'image' etc)? Possibly even move some information from the title into the body, so it instead of, "Bob in The Room", "Message", it would be, "The Room", "Bob sent a message" or something.

Also, although I don't think you can turn notifications on for them in Riot, this would also break notifications for other events like joins/parts/room name changes etc.